src/util/moveItemInArray.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Shifts element array to designated position in array. fromIndex
+ * is the chosen array element and toIndex is the array position for
```

Comment:
this is already written in the param description...

Reply:
removed